src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -64,7 +64,10 @@ public function guessRequiredProvider()
         return array(
             array(new NotNull(), new ValueGuess(true, Guess::HIGH_CONFIDENCE)),
             array(new NotBlank(), new ValueGuess(true, Guess::HIGH_CONFIDENCE)),
-            array(new True(), new ValueGuess(true, Guess::HIGH_CONFIDENCE)),
+            if (PHP_VERSION_ID < 70000) {
```

Comment:
Parse error here. Le legacy test should be split in a dedicated `public function testLegacyGuessRequired()` test method, with the `@group legacy` annotation.